pages/common/atoum.md

```diff
@@ -0,0 +1,36 @@
+# atoum
+
+> A simple, modern and intuitive unit testing framework for PHP.
+> Homepage: <https://atoum.org>.
```

Comment:
Sorry, for the late review. I wrote it a day before but it was not submitted and stayed as "pending".

Reply:
Great catch, @Aracki! Though I can't help but think that this might be an issue for the project owners who really should support HTTPS in this day & age lol

I agree, we should update the tldr-page to include an actually working URL for now.

Reply:
It does support HTTPS, it only has an expired certificate. That's bad, of course, but they'll probably update it.

Reply:
Weird, I wasn't getting an error before. Would be good to update this for now though. :+1:

Reply:
They use a gh-pages repo for their site, in fact the certificate it tries to use if you conenct to HTTPS is from GitHub, but of course since the domain is different it does not work. I'm not sure if they ever intended to use HTTPS to begin with, after all it's only a documentation website, it doesn't *really* need HTTPS. 

![image](https://user-images.githubusercontent.com/14198070/57875767-fa444280-7813-11e9-8052-fb041e436a8b.png)

Reply:
Hrm. These days, there are may more reasons to enable https than just protecting personal details. Surprisingly, the US government [have an article on the subject](https://https.cio.gov/everything/), @mebeim